de.dlr.sc.virsat.model.extension.cefx/concept/concept.concept

```diff
@@ -1,179 +1,179 @@
+		Cardinality 1;
+				
+		Type massLaunch of Category Parameter quantityKind "Mass" unit "Kilogram";
+		Type massTotal of Category Parameter quantityKind "Mass" unit "Kilogram";
+		Type massTotalWithMargin of Category Parameter quantityKind "Mass" unit "Kilogram";
+		Type massTotalWithMarginWithSystemMargin of Category Parameter quantityKind "Mass" unit "Kilogram";
+		Type massAdapter of Category Parameter quantityKind "Mass" unit "Kilogram";
+		Type massPropellant of Category Parameter quantityKind "Mass" unit "Kilogram";
+		Type massLaunchMax of Category Parameter quantityKind "Mass" unit "Kilogram";
+		Type massBuffer of Category Parameter quantityKind "Mass" unit "Kilogram";
+		
+		Ref: massTotal = summary{SubSystemMassParameters.massTotal};
+		Ref: massTotalWithMargin = summary{SubSystemMassParameters.massTotalWithMargin};
+		Ref: massTotalWithMarginWithSystemMargin = massTotalWithMargin * (1 + SystemParameters.systemMargin);
+		Ref: massLaunch = massTotalWithMarginWithSystemMargin + massPropellant + massAdapter;
+		Ref: massBuffer = massLaunchMax - massLaunch;
+	}
+	
+	Category SystemPowerParameters {
+		Applicable For [de.dlr.sc.virsat.model.extension.ps.ConfigurationTree, de.dlr.sc.virsat.model.extension.ps.AssemblyTree, de.dlr.sc.virsat.model.extension.ps.ElementConfiguration, de.dlr.sc.virsat.model.extension.ps.ElementOccurence];
+		Cardinality 1;
+				
+		Type powerAvgWithMargin of Category Parameter quantityKind "Power" unit "Watt";
+		Type powerAvgWithSystemMargin of Category Parameter quantityKind "Power" unit "Watt";
+		Type powerEnergyWithMargin of Category Parameter quantityKind "Energy and Work" unit "Joule";
+		Type powerEnergyWithSystemMargin of Category Parameter quantityKind "Energy and Work" unit "Joule";
+		
+		Ref: powerAvgWithMargin = summary{SubSystemPowerParameters.powerAvgWithMargin};
+		Ref: powerAvgWithSystemMargin = powerAvgWithMargin * (1 + SystemParameters.systemMargin);
+		Ref: powerEnergyWithMargin = summary{SubSystemPowerParameters.powerEnergyWithMargin};
+		Ref: powerEnergyWithSystemMargin = powerEnergyWithMargin * (1 + SystemParameters.systemMargin);
+	}
+	
+	/** SubSystem level specific collections of data and parameters */
+	
+	Category SubSystemMassParameters {
+		Applicable For [de.dlr.sc.virsat.model.extension.ps.ElementConfiguration, de.dlr.sc.virsat.model.extension.ps.ElementOccurence, de.dlr.sc.virsat.model.extension.ps.ElementRealization];
+		Cardinality 1;
+				
+		Type massTotal of Category Parameter quantityKind "Mass" unit "Kilogram";
+		Type massTotalWithMargin of Category Parameter quantityKind "Mass" unit "Kilogram";
+		Type massMarginPercentage of Category Parameter quantityKind "Dimensionless" unit "Percent";
+		Type massMargin of Category Parameter quantityKind "Mass" unit "Kilogram";
+		
+		Ref: massTotal = summary{EquipmentMassParameters.massTotal};
+		Ref: massTotalWithMargin = summary{EquipmentMassParameters.massTotalWithMargin, 1};
```

Comment:
And we also have to do it for the values with margin... massTotalWithMargin and also same for power values...